vme/src/account.cpp

```diff
@@ -654,7 +654,7 @@ void do_account(unit_data *ch, char *arg, const command_info *cmd)
         return;
     }
 
-    c = skip_spaces(c);
+    c = (char *) skip_spaces(c);
```

Comment:
I think maybe `c` could be const at 639 - and possible to do in the other functions maybe. So less casting

Reply:
Tried, it complains in find_unit :-) find_unit could probably be changed to const .,.. but it's a rat hole...